frame/chainrelay/eth/relay/src/tests.rs

```diff
@@ -1,175 +1,159 @@
 //! Tests for eth-relay.
 
-use std::str::FromStr;
-
-use eth_primitives::{
-	receipt::{LogEntry, TransactionOutcome},
-	Bloom, EthAddress, H64, U128,
-};
-use frame_support::assert_ok;
+use eth_primitives::{receipt::TransactionOutcome, U128};
+use frame_support::{assert_err, assert_ok};
 use frame_system::RawOrigin;
-use hex_literal::hex;
```

Comment:
```
error: cannot find macro `hex` in this scope
   --> frame/chainrelay/eth/relay/src/tests.rs:181:15
    |
181 |             hash: Some(hex!("034cd83d9150808de742592f57e302dc9eccc71af270639dc5f236e5bdd7d3e3").into()),
    |                        ^^^
    |
    = help: have you added the `#[macro_use]` on the module/import?
error: cannot find macro `hex` in this scope
   --> frame/chainrelay/eth/relay/src/tests.rs:179:28
    |
179 |                 rlp::encode(&H64::from(hex!("a98d18400422c64e"))),
```

Reply:
This is caused by the incorrect merge of the latest commit.

Reply:
Oops, still have compile error.

Reply:
It's strange, the error part shoud have been removed alreadly, I'll find out why travis running the old code tonight.

Reply:
Travis passed the tests on my [fork][0], but not at this pr.

[0]: https://github.com/clearloop/darwinia